WTWSMS/common/religions/00_religions.txt

```diff
@@ -1,5428 +1,5635 @@
-christian = {
```

Comment:
That was an earlier (now resolved) issue where I accidently saved "RELIGIONS.TXT" as an Unicode file instead of an ANSI file. I have since corrected that, so there is no need to worry.